assets/js/templates/student_sidebar_template.html

```diff
@@ -0,0 +1,18 @@
+<div class="panel-group" id="accordion">
+  <div class="panel panel-default" ng-repeat="module in library">
+      <div class="panel-heading">
+          <h4 class="panel-title">
+              <a data-toggle="collapse" data-target="#collapse{{module.id}}" data-parent="#accordion" href="">{{module.name}}</a>
+          </h4>
+      </div>
+      <div id="collapse{{module.id}}" class="panel-collapse collapse">
+          <div class="panel-body">
+              <ul>
+                <li ng-repeat="lesson in module.lessons">
+                  <a href="/#/library/{{module.library.id}}/module/{{module.id}}/lesson/{{lesson.id}}">{{lesson.title}}</a>
+                </li>
+              </ul>
+          </div>
+      </div>
```

Comment:
check your spacing.. it looks like some are 2 spaced tabs and some are 4 spaced tabs